x/posts/spec/04-messages.md

```diff
@@ -11,7 +11,7 @@ slug: messages
 A post can be created using the `MsgCreatePost`.
 
 ```js reference
-https://github.com/desmos-labs/desmos/blob/v4.1.0/proto/desmos/posts/v2/msgs.proto#L36-L89
+https://github.com/desmos-labs/desmos/blob/master/proto/desmos/posts/v2/msgs.proto#L36-L89
```

Comment:
I've updated all the specs to refer to the `master` branch instead of the `v4.1.0` tag, since there might have been some changes inside that branch since the last release (like in this case). It's also always better to reference the `master` branch instead of a version, since these specs will leave inside the `master` branch itself. We can always change the referenced blob when we publish the docs anyway